utils/markdown/renderer.jsx

```diff
@@ -120,19 +119,43 @@ export default class Renderer extends marked.Renderer {
                 }
             }
         }
-        src = PostUtils.getImageSrc(src, this.formattingOptions.proxyImages);
-        let out = '<img src="' + src + '" alt="' + text + '"';
-        if (title) {
-            out += ' title="' + title + '"';
-        }
-        if (dimensions.length > 0) {
-            out += ' width="' + dimensions[0] + '"';
-        }
-        if (dimensions.length > 1) {
-            out += ' height="' + dimensions[1] + '"';
+
+        //After looking into the mark package, there appears to be no in-line level render function in marked.js
+        //to render a video. Since we just want to render gifv files like we do gifs, we can check if
+        //the URL src is a gifv, then render it in a <video> HTML5 element rather than an <img> element without
+        //having to touch the mark package. There is some DRY issues here, but I am not sure how to overcome that.
```

Comment:
Please update into a concise comment. Might be good to move above comments to PR description.